warcry_proj/warcry_proj/static/theme.css

```diff
@@ -0,0 +1,154 @@
+@import url(//fonts.googleapis.com/css?family=Open+Sans:300,400,600,700);
```

Comment:
A lot of these templates might be covered for you when you switch over to use Zappa. I'm not 100% sure though. Certainly might save you some work here.